### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by lannisterrr**
Added - Screens and Modals for the layout

### CODE REVIEW

File: src/components/AddTask.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { BsPlus, BsFillTrashFill } from 'react-icons/bs';
+import { FiEdit } from 'react-icons/fi';
+import Modal from './Modal';
+function AddTask() {
+  const [showModal, setShowModal] = useState(false);
+  const handleShowModal = toggle => {
+    setShowModal(toggle);
```
Comment (GuptaEesh) [inline src/components/AddTask.js:8]: Could have directly done !showModal instead of passing toggle everytime

**Review by GuptaEesh — COMMENTED** [review]
The setup looks clean and good to merge.
Added a comment please look into it for making your code less verbose

File: src/components/AddTask.js (hunk 1 above)
Comment (ashishnnnnn) [reply src/components/AddTask.js:8]: Yaa.. could be shown..
Looks like unnecessary state managements

File: src/components/AddTask.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  };
+  return (
+    <>
+      <div className="w-screen flex-col justify-center items-center min-h-screen bg-primary">
+        <h1 class="text-3xl md:text-5xl p-4 font-normal leading-normal mt-0 mb-2 text-secondary text-center">
```
Comment (ashishnnnnn) [inline src/components/AddTask.js:13]: Here It should be class ..

**Review by ashishnnnnn — APPROVED** [review]
Code seems pretty much fine..
Apart from the suggestion

**Pull request merged**